### ISSUE TITLE
Make ALT text length configurable

### ISSUE THREAD

**Issue by bagofarms** [ucfopen/UDOIT#304]
This value should be set to 100 by default, and should apply for both detection of ALT text that is too long and for the UFIXIT for ALT text.

**Comment by bagofarms** [comment]
On second thought, make the default 125 characters.
